docs/install-and-setup.md

```diff
@@ -25,15 +25,15 @@ calendar] to know how to do it).
 
 ## Create the script
 
-Copy the whole content of [this file][Main code file].  
+Copy the whole content of [this file][Main code file].
```

Comment:
removing those 2 spaces removed the new line in markdown. are you sure that's intended? it applies to lines below as well

Reply:
my vim is set to remove trailing spaces, will manually check the file and all changes